src/FFmpegReader.cpp

```diff
@@ -995,6 +1012,7 @@ std::shared_ptr<Frame> FFmpegReader::ReadStream(int64_t requested_frame) {
 			end_of_file = true;
 			break;
 		}
+        attempts++;
```

Comment:
I saw one `continue` statement in this loop. Would it be better to increment attempts at the top of the loop to guarantee we're moving toward breaking out of the loop?